src/__tests__/definitions/invalid-map-ob-link.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Comment": "An example of the Amazon States Language using a map state to map over items in parallel.",
+  "StartAt": "Map",
+  "States": {
+    "Map": {
+      "Type": "Map",
+      "Next": "Final State",
+      "InputPath": "$.input",
+      "ItemsPath": "$.items",
+      "MaxConcurrency": 0,
+      "Iterator": {
+        "StartAt": "ChoiceState",
+        "States": {
+          "ChoiceState": {
+            "Type": "Choice",
+            "Choices": [
+              {
+                "Variable": "$.foo",
+                "NumericEquals": 1,
+                "Next": "Wait 20s"
+              },
+              {
+                "Variable": "$.foo",
+                "NumericEquals": 2,
+                "Next": "Final State"
```

Comment:
the link to `Final State` here is invalid since the target state is defined outside of the map's container

Reply:
could you add this comment in the "Comment" filed, line 2?